examples/go.mod

```diff
@@ -135,7 +135,7 @@ require (
 	github.com/pkg/term v1.1.0 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/pulumi/esc v0.5.6 // indirect
-	github.com/pulumi/pulumi/sdk/v3 v3.90.1 // indirect
+	github.com/pulumi/pulumi/sdk/v3 v3.91.0 // indirect
```

Comment:
I wonder it made this change but I guess replace takes care of it anyway. Perhaps tidy is not good after a replace? Not sure; we do want to chase down minor indirect dependencies automatically though. This is probably as good as it gets. CC @iwahbe

Reply:
If we don't tidy then `go` complains that it can't parse the `go.mod` file, since most of `go`'s resolvers don't handle SHAs as versions correctly.

I think this is good enough.